src/Game.cpp

```diff
@@ -205,8 +205,8 @@ void Game::ToJson(Json &jsonObj)
 	jsonObj["hyperspace_end_time"] = m_hyperspaceEndTime;
 
 	// Delete camera frame from frame structure:
-	if (m_gameViews->m_worldView->GetCameraContext()->GetCamFrame())
-		m_gameViews->m_worldView->EndCameraFrame();
+	bool have_cam_frame = m_gameViews->m_worldView->GetCameraContext()->GetCamFrame().valid();
```

Comment:
By the by, the FrameId struct has an implicit conversion to `bool`. So it's not required to put the `.valid()` on there, unless you want it for code clarity :D.

Reply:
Yes, you're right :P .., On "the Book" too" :D